File: app/src/main/java/org/deiverbum/app/model/LiturgyTime.java
package org.deiverbum.app.model;

/**
 * <p>Clase para expresar los tiempos litúrgicos.</p>
 * @author A. Cedano
 * @version 1.0
 * @since 2022.2
 */
public class LiturgyTime {

    private Integer tiempoId;
    private String tiempo;
    private String liturgyName;

    public Integer getTiempoId() {
        return tiempoId;
    }

    public void setTiempoId(Integer tiempoId) {
        this.tiempoId = tiempoId;
    }

    public String getTiempo() {
        return tiempo;
    }

    public void setTiempo(String tiempo) {
        this.tiempo = tiempo;
    }

    public String getLiturgyName() {
        return liturgyName;
    }

    public void setLiturgyName(String liturgyName) {
        this.liturgyName = liturgyName;
    }
    public String getAllForRead() {
        return getLiturgyName();
    }

    }

File: app/src/main/java/org/deiverbum/app/model/LiturgyTime.kt
package org.deiverbum.app.model

/**
 *
 * Clase para expresar los tiempos litúrgicos.
 *
 * @author A. Cedano
 * @version 1.0
 * @since 2023.1
 */
class LiturgyTime {
    var timeID: Int = 0
    var timeName: String? = null
    var liturgyName: String? = null
    val allForRead: String?
        get() = liturgyName
}